tests/PHPStan/Analyser/data/bug-7490.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types=1);
+
+namespace Bug7490;
+
+use function PHPStan\Testing\assertType;
+
+function () {
+	assertType('*ERROR*', 1 << -1);
+	assertType('*ERROR*', 1 >> -1);
+};
```

Comment:
I've used a simpler test case, because the test case in the issue took around a minute to process. This tests exactly the same thing.